app/controllers/devise_registration_controller.rb

```diff
@@ -67,25 +104,6 @@ def phone_code
     redirect_to url_for_state and return unless registration_state.state == "phone"
   end
 
-  def phone_code_send
```

Comment:
What about resending the MFA code during registration?

Reply:
I forgot we're using this method for resending the codes, I'll build it into the `phone_resend` method, since that feels like a more appropriate place now.